### PR TITLE
Update self-hosted docs for helm deprecation

### PR DESCRIPTION
**Pull request by atduarte**
## Description

Update self-hosted docs to make clear that Helm is now deprecated, and the new Installer or Replicated should be used. Without affecting SEO.

[Related PR](https://github.com/gitpod-io/website/pull/1441)

## Release Notes
```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1536"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1536"></a>

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -62,8 +62,9 @@
 /docs/python-in-gitpod /docs/languages/python 301
 /docs/r-in-gitpod /docs/languages/r 301
 /docs/ruby-in-gitpod /docs/languages/ruby 301
-/docs/rust-in-gitpod /docs/languages/rust 301
+/docs/rust-in-gitpod /docs/languages/rust 30
```
Comment (MrSimonEmms) [inline _redirects:65]: Should this be `301`?

**Review by MrSimonEmms — COMMENTED** [review]
I think that'll work for the time being.

Do we want to change the paths from `self-hosted/latest` to `self-hosted/helm` (or similar) to allow us to have `self-hosted/latest` to be the latest version of the Installer?

**Comment by atduarte** [comment]
> Do we want to change the paths from self-hosted/latest to self-hosted/helm (or similar) to allow us to have self-hosted/latest to be the latest version of the Installer?

Not sure I understood, @MrSimonEmms. With this change `self-hosted/latest` lists the various available methods (installer, replicated and the deprecated helm). You may check it out [here](https://deploy-preview-1536--gitpod-kumquat.netlify.app/docs/self-hosted/latest).

File: gitpod/docs/self-hosted/latest/requirements/index.md [hunk 2]
```diff
@@ -15,10 +15,10 @@ This page details the software and hardware requirements for installing Gitpod S
 
 Gitpod Self-Hosted runs well on:
 
-- [Amazon Elastic Kubernetes Service](./installation/on-amazon-eks)
-- [Google Kubernetes Engine](./installation/on-gke)
-- [K3s](./installation/on-kubernetes)
-- [Microsoft Azure Kubernetes Service](./installation/on-microsoft-aks)
+- Amazon Elastic Kubernetes Service]
```
Comment (MrSimonEmms) [inline gitpod/docs/self-hosted/latest/requirements/index.md:18]: Trailing `]`

**Review by MrSimonEmms — APPROVED** [review]

**Review by mikenikles — APPROVED** [review]
👍

**Pull request merged**